projects/storefrontstyles/scss/components/content/navigation/_my-account-v2-navigation.scss

```diff
@@ -53,3 +56,26 @@
     display: inline-block;
   }
 }
+
+@media screen and (max-width: 544px) {
+  cx-navigation-ui h4,
+  span {
+    width: 100% !important;
+    font-size: 14px;
+  }
```

Comment:
- We may need to keep these within the component's scope. 
- There should be a better way to overwrite these styles then using the nuclear option of `!important`.
- We probably should stick to the breakpoints established by Spartacus already.